uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/DebuggerTest.java

```diff
@@ -196,12 +195,22 @@ public void testDebugger() {
 		AbstractWait.sleep(TimePeriod.getCustom(5));
 		assertEquals(CHOICE_ID, variables.getValue("Endpoint"));
 
-		// get body of message
-		variables.close();
-		AbstractWait.sleep(TimePeriod.SHORT);
-		variables.open();
-		new DefaultTree().getItems().get(4).getItems().get(0).select();
-		assertTrue(new DefaultStyledText().getText().contains("<name>Erie Zoo</name>"));
+		for (int i = 0; i < 4; i++) {
+			variables.close();
+			AbstractWait.sleep(TimePeriod.SHORT);
+			variables.open();
+			new DefaultTree().getItems().get(4).getItems().get(0).select();
```

Comment:
uff... what's this?

Reply:
in the future it would be nice to have a comment for something like this ;-)
```
new DefaultTree().getItems().get(4).getItems().get(0).select();
```